pkg/controller/daemon/daemon_controller.go

```diff
@@ -262,8 +285,9 @@ func (dsc *DaemonSetsController) Run(ctx context.Context, workers int) {
 
 	defer dsc.queue.ShutDown()
 
-	klog.Infof("Starting daemon sets controller")
-	defer klog.Infof("Shutting down daemon sets controller")
+	logger := klog.LoggerWithName(klog.FromContext(ctx), "DaemonSet")
+	logger.Info("Starting daemon sets controller")
```

Comment:
If the logger is named, perhaps this could be

```suggestion
	logger.Info("Starting controller")
```